gitlabform/configuration/core.py

```diff
@@ -162,13 +163,64 @@ def is_group_skipped(self, group) -> bool:
     def is_skipped(self, an_array: list, item: str) -> bool:
         pass
 
+    @staticmethod
+    def validate_break_inheritance_flag(config, **kwargs):
+        has_invalid_flag = False
+        level = None
+        for key, value in config.items():
+            if "inherit" == key:
+                if kwargs["parent"] == "empty" and (
+                    kwargs["level"] == "group"
+                    or kwargs["level"] == "project"
+                    or kwargs["level"] == "subgroup"
+                ):
+                    level = kwargs["level"]
+                    has_invalid_flag = True
+                elif kwargs["level"] == "common":
+                    level = "common"
+                    has_invalid_flag = True
+            if has_invalid_flag:
+                fatal(
+                    "The inheritance break flag cannot be placed at the {config_level} level\n"
+                    "because {config_level} level is the highest level in the configuration file.\n".format(
+                        config_level=level
+                    ),
+                    exit_code=EXIT_PROCESSING_ERROR,
```

Comment:
This is an input error, not a processing error.

I added commit to clarify when each exit code should be used here https://github.com/gdubicki/gitlabform/pull/339/commits/554be4dfb807d7a0ab963dfdc51dc06bd11ecc3f . Is this information enough?

Reply:
Yup, it's enough, thanks!